rpc/api/eth.go

```diff
@@ -166,6 +167,20 @@ func (self *ethApi) IsMining(req *shared.Request) (interface{}, error) {
 	return self.xeth.IsMining(), nil
 }
 
+func (self *ethApi) IsSyncing(req *shared.Request) (interface{}, error) {
```

Comment:
The name in the rpc is `eth_syncing` and returns also a boolean or the stats. I don't know a better name and think this name is understandable, otherwise we would need to methods :)